doc/jsk_perception/nodes/vqa_node.md

```diff
@@ -0,0 +1,91 @@
+# vqa_node.py
+![][images/vqa.png]
+
+The ROS node for Visual Question Answering(VQA).
+
+## System Configuration
+![][images/large_scale_vil_system.pdf]
```

Comment:
Same as above.

If we want to show a system configutation figure directly on Readme.md, we should replace pdf file to some other image file formats (jpeg or png).